indexing-service/src/main/java/org/apache/druid/indexing/common/task/NoopTask.java

```diff
@@ -29,33 +29,38 @@
 import org.apache.druid.indexing.common.config.TaskConfig;
 import org.apache.druid.java.util.common.DateTimes;
 import org.apache.druid.java.util.common.StringUtils;
+import org.apache.druid.java.util.emitter.service.ServiceMetricEvent;
 import org.apache.druid.server.security.ResourceAction;
 
 import javax.annotation.Nonnull;
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Set;
 import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
  */
 public class NoopTask extends AbstractTask implements PendingSegmentAllocatingTask
 {
   public static final String TYPE = "noop";
+  public static final String NOOP_TASK_EVENT_STARTED = "noopTask-started";
   private static final int DEFAULT_RUN_TIME = 2500;
 
   @JsonIgnore
   private final long runTime;
 
+  @JsonIgnore
```

Comment:
Nit: I think this annotation is not required. Any field/method not explicitly marked `@JsonProperty` is skipped from serialization by default.